java/src/shared/model/player/Player.java

```diff
@@ -702,6 +702,13 @@ public void setDiscarded(boolean discarded) {
         this.discarded = discarded;
     }
 
+    public void setVictoryPoints(int points){
+        assert points >=0;
+        assert  points <=10;
```

Comment:
Also on that note, perhaps it'd be better to have an `incrementVictoryPoints` instead. Using a setter requires the caller to know how many total points the player will now have, which involves calling `getVictoryPoints`, adding to it, then setting that sum as the new victory points. It'd be much cleaner to have something like `player.incrementPoints(2)`. I feel it'd be a much more common use-case to just increment the points rather than set it to an entirely new value.